zips/zip-0000.rst

```diff
@@ -693,6 +747,11 @@ will specify a network upgrade in which the consensus change is to activate.
 A ZIP's status is Released if it is Proposed, Active, Implemented, or Final
 (i.e. not Draft, Rejected, Obsolete, or Withdrawn).
 
+A ZIP that has significant security or privacy implications MUST NOT
+be changed from a non-Released status to a Released status without the
+design having undergone independent security review from qualified
+security or privacy experts.
```

Comment:
This potentially leaves a loophole where a change is made after a ZIP is Released that is substantive enough to require another independent security review. Typically I think we've handled that in the past by including all such changes made in a particular upgrade in the upgrade's design review, but the ZIP process should require that, I think.

These aren't sufficient to catch all such cases:
> * it disregards compatibility with the existing Zcash blockchain or ecosystem;
> * in the case of a Proposed, Active, Implemented, or Final ZIP, the update makes a substantive change to which there is significant community opposition;
> * an update to an existing ZIP extends or changes its scope to an extent that would be better handled as a separate ZIP;